### PR TITLE
[CapitalOne.com] New ruleset

### PR DESCRIPTION
**Pull request by chris-barry**

### CODE REVIEW

**Comment by J0WI** [comment]
Please have a look at our ruleset style guide where we explain how to find more subdomains.

**Comment by jeremyn** [comment]
Ruleset style guide [here](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md).

Example second query [here](https://www.google.com/search?q=site%3A*.capitalone.com+-site%3Awww.capitalone.com).

**Comment by chris-barry** [comment]
On it :+1:

File: src/chrome/content/rules/CapitalOne.com.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	- press
+	- www.application
+-->
+<ruleset name="CapitalOne.com">
+	<target host="capitalone.com" />
+	<target host="www.capitalone.com" />
+	<target host="m.capitalone.com" />
+
+	<target host="ane.capitalone.com" />
+	<target host="application1.capitalone.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: also application. and application2

File: src/chrome/content/rules/CapitalOne.com.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	<target host="apply.capitalone.com" />
+	<target host="campus.capitalone.com" />
+	<target host="community.capitalone.com" />
+	<target host="creditwise.capitalone.com" />
+	<target host="dealernavigator.capitalone.com" />
+	<target host="developer.capitalone.com" />
+	<target host="findmycard.capitalone.com" />
+	<target host="frtextmessaging.capitalone.com" />
+	<target host="frapplication2.capitalone.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: frapplication.capitalone.com
frapplication1.capitalone.com

File: src/chrome/content/rules/CapitalOne.com.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	<target host="hlblog.capitalone.com" />
+	<target host="jobs.capitalone.com" />
+	<target host="loanhub.capitalone.com" />
+	<target host="loanhub2.capitalone.com" />
+	<target host="login1.capitalone.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: login.capitalone.com
login2.capitalone.com

File: src/chrome/content/rules/CapitalOne.com.xml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	<target host="mortgages.capitalone.com" />
+	<target host="preferences.capitalone.com" />
+	<target host="services2.capitalone.com" />
+	<target host="servicing.capitalone.com" />
+	<target host="sparkbusiness.capitalone.com" />
+	<target host="travelrewardspn.capitalone.com" />
+	<target host="verified.capitalone.com" />
+	<target host="wwwbeta2.capitalone.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: securedcards.capitalone.com
sakscanada.capitalone.com
hudsonsbay.capitalone.com
p2ppayments.capitalone.com
fssso.capitalone.com
fssso1.capitalone.com
fssso2.capitalone.com
content.capitalone.com
frservicing.capitalone.com

File: src/chrome/content/rules/CapitalOne.com.xml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+	<target host="frtextmessaging.capitalone.com" />
+	<target host="frapplication2.capitalone.com" />
+	<target host="hlblog.capitalone.com" />
+	<target host="jobs.capitalone.com" />
+	<target host="loanhub.capitalone.com" />
+	<target host="loanhub2.capitalone.com" />
+	<target host="login1.capitalone.com" />
+	<target host="mortgages.capitalone.com" />
+	<target host="preferences.capitalone.com" />
+	<target host="services2.capitalone.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: services.capitalone.com

File: src/chrome/content/rules/CapitalOne.com.xml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+	<target host="community.capitalone.com" />
+	<target host="creditwise.capitalone.com" />
+	<target host="dealernavigator.capitalone.com" />
+	<target host="developer.capitalone.com" />
+	<target host="findmycard.capitalone.com" />
+	<target host="frtextmessaging.capitalone.com" />
+	<target host="frapplication2.capitalone.com" />
+	<target host="hlblog.capitalone.com" />
+	<target host="jobs.capitalone.com" />
+	<target host="loanhub.capitalone.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: loanhub1.capitalone.com

**Comment by J0WI** [comment]
For some reason they have many [12] aliases.

Please also have a look at the failing tests on Travis. You may need to include valid test urls for these hosts and exclude the root path (`/$`).

**Comment by chris-barry** [comment]
I moved some of the 404ing ones to the notes. I'm gonna see how Travis likes it, my local machine won't install the deps (I'll have to look in to that).

**Comment by J0WI** [comment]
LGTM, thanks for adding them.

**Pull request merged**